ui/src/components/Tabs/TimeTrackingTab/IndividualTimeModal.js

```diff
@@ -1,114 +1,84 @@
 import React, { useContext, useState } from "react";
 
-import { Button, Divider, Icon, Modal, ModalActions } from "semantic-ui-react";
-import { formatDate, formatDateTime } from "../../util/functions/utils";
+import { Button, Icon } from "semantic-ui-react";
+// import { formatDate, formatDateTime } from "../../util/functions/utils";
 import { SecureFetch } from "../../util/functions/secureFetch";
-import InnerHTML from "dangerously-set-html-content";
+// import InnerHTML from "dangerously-set-html-content";
 import { UserContext } from "../../util/functions/UserContext";
 import { config } from "../../util/functions/constants";
+import ModalWrapper from "../../shared/ModalWrapper";
+import IndividualTimeModalContent from "./IndividualTimeModalContent";
 
 export default function IndividualTimeModal(props) {
   const [open, setOpen] = useState(false);
-  const [submission, setSubmission] = useState({});
-  const [files, setFiles] = useState([]);
-  const [noSubmission, setNoSubmission] = useState(true);
-  const [due, setDue] = useState();
-  const [late, setLate] = useState(false);
-  const [day, setDay] = useState(0);
+  // const [submission, setSubmission] = useState({});
+  // const [files, setFiles] = useState([]);
+  // const [noSubmission, setNoSubmission] = useState(true);
+  // const [due, setDue] = useState();
+  // const [late, setLate] = useState(false);
+  // const [day, setDay] = useState(0);
   const { user } = useContext(UserContext);
```

Comment:
Naming this variable `user` when there is a property called `user` may be confusing.
```suggestion
  const { userContext } = useContext(UserContext);
```